content/rosa/entra-external-auth/index.md

```diff
@@ -0,0 +1,467 @@
+
+```bash
+apiVersion: v1
+clusters:
+- cluster:
+    server: ${API_URL}
+  name: cluster
+contexts:
+- context:
+    cluster: cluster
+    namespace: default
+    user: oidc
+  name: admin
+current-context: admin
+kind: Config
+preferences: {}
+users:
+- name: oidc
+  user:
+    exec:
+      apiVersion: client.authentication.k8s.io/v1
+      args:
+      - oidc-login
+      - get-token
+      - --oidc-issuer-url=https://login.microsoftonline.com/${TENANT_ID}/v2.0
+      - --oidc-client-id=${CLIENT_ID}
+      - --oidc-client-secret=${CLIENT_SECRET}
+      - --oidc-extra-scope=email
+      - --oidc-extra-scope=openid
+      - --grant-type=device-code
+      - --skip-open-browser
+      - --token-cache-storage=disk
+      command: kubectl
+      env: null
+      interactiveMode: Never
+```
+Set the KUBECONFIG environment variable to the location of the rosa-cluster.kubeconfig file which will enable the OpenShift CLI to authenticate against the ROSA cluster with the contents of the file.
+
+To confirm that the identity details from the OIDC token is being used by the OpenShift Command Line Interface:
+
+```bash
+export KUBECONFIG=$(pwd)/rosa-auth-headless.kubeconfig
+```
+
+Confirm access to cluster
+
+```bash
+oc get nodes
+```
+example output:
+
+```bash
+Please enter the following code when asked in your browser: XXXXXXX
+Please visit the following URL in your browser: https://microsoft.com/devicelogin
+
+ATTRIBUTE   VALUE
+Username    xxxxxx@redhat.com
+Groups      [000000000000000000 system:authenticated]
+NAME                         STATUS   ROLES    AGE     VERSION
+ip-10-0-0-170.ec2.internal   Ready    worker   3h29m   v1.30.7
+ip-10-0-1-171.ec2.internal   Ready    worker   3h30m   v1.30.7
+ip-10-0-2-161.ec2.internal   Ready    worker   3h29m   v1.30.7
+```
+To verify logged in as user of group run
+
+```bash
+oc auth whoami
+```
+```bash
+Please enter the following code when asked in your browser: XXXXXX
+Please visit the following URL in your browser: https://microsoft.com/devicelogin
+ATTRIBUTE   VALUE
+Username    XXXXXXX@redhat.com
+Groups      [0000000000000000 system:authenticated]
+```
+
+### Revoke break glass credential
+Upon completion of any activities that require elevated access to the ROSA cluster (such as the management of RBAC policies) using a break glass credential, all remnants of its use should be removed. Proper removal ensures that no unexpected actions are performed using these privileged credentials. 
+There are two locations where the management of this credential should be performed:
+ROSA Cluster
+kubeconfig file
+
+Delete the kubeconfig file
+rm $KUBECONFIG
+
+A break glass credential can be removed from a ROSA cluster using the `rosa revoke break-glass-credentials` command. Execute the following command to revoke the break glass credential associated with a ROSA cluster.
+
+```bash
+rosa revoke break-glass-credentials -c ${ROSA_CLUSTER_NAME}
+```
```

Comment:
Instead of writing this here, explain the importance of revoking a break glass credential and point them to the OpenShift documentation on how to do so.